testscript/testdata/ui-home.txtar

```diff
@@ -0,0 +1,205 @@
+# vi: set ft=conf
+
+# convert crlf to lf on windows
+[windows] dos2unix home-in.txt home.txt about-in.txt about.txt
+
+# test repositories tab
+ui home-in.txt
+cmp stdout home.txt
+
+# test about tab
+ui about-in.txt
+cmp stdout about.txt
+
+# add a new repo
+soft repo create .soft-serve -n 'Config' -d '"Test Soft Serve"'
+soft repo description .soft-serve
+stdout 'Test Soft Serve'
+soft repo project-name .soft-serve
+stdout 'Config'
+
+# clone repo
+git clone ssh://localhost:$SSH_PORT/.soft-serve config
+
+# create readme file
+mkfile ./config/README.md '# Hello World\nTest Soft Serve'
+git -C config add -A
+git -C config commit -m 'Initial commit'
+git -C config push origin HEAD
+
+# test repositories tab
+ui home-in.txt
+cmpenv stdout home2.txt
+
+# test about tab
+ui about-in.txt
+cmp stdout about2.txt
+
+-- home-in.txt --
+    q
+-- home.txt --
+[?25l[?1049h[2J[1;1H[1;1H[?25l[?1002h                                                                                

```

Comment:
I think we could force ASCII mode on tests to make the output more readable (and less fragile) 🤔